src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -375,9 +375,18 @@ public function testFilesExists()
         mkdir($basePath);
         touch($basePath.'file1');
         mkdir($basePath.'folder');
+        $this->assertTrue($this->filesystem->exists($basePath.'file1', false));
+        $this->assertTrue($this->filesystem->exists($basePath.'folder', false));
```

Comment:
i'd remove `, false` here, to ensure we test the default behavior

Reply:
done